### DISCUSSION TITLE
Adding PRONOM Classification in siegfried output

### DISCUSSION THREAD

**Discussion by robin-francois** [richardlehane/siegfried#207] [Ideas]
Hello Siegfried community!

I have exchanged some email with @richardlehane who as invited me to open a GitHub discussion, so here I am.

I am currently working with our internal development team on building up the digital preservation platform of our institution. We need to identify aggregate file formats so that we can "deaggregate" them and to process each individual file separately.

If Siegfried could output the PRONOM Classification/FormatTypes, it would allow using the output to take a decision.

 As pointed out by @richardlehane, the current `sets` could also be used to perform so kind of filtering/grouping. But for some reason, it appears less natural to me that having the information output and perform the filtering based on the output.

**Comment by ross-spencer** [comment]
+1 from me. I mooted as much in a recent Wikidata paper/presentation. DROID has an equivalent feature request: https://github.com/digital-preservation/droid/issues/663 and personally I'd love to incorporate this data into Demystify!

**Reply by thorsted** [reply]
+1 for me as well. If PRONOM can add this to their signature file for each format, this would enhance our Rosetta Format Library immensely.

**Comment by ross-spencer** [comment]
Folks following along here, there's a PR against this now: https://github.com/richardlehane/siegfried/pull/209

There's a small question about naming the field, which I'll use the discussion here to share with people on social media. 

`TYPE` is used in DROID's output, this is the label I naively used, but this is easily changed. I just wanted to clarify with the community what may be the best option. 

> LoC uses "content categories" to describe this: https://www.loc.gov/preservation/digital/formats/content/content_categories.shtml
>
> PRONOM as we know uses "Classification".
>
> Wikidata doesn't seem to have the equivalent predicate, it tends towards instance of, and use/used for to describe similar. There may be an equivalent predicate.

From Robin too:

> Wikipedia seems to use type of format in some pages. I have no favoured term. Both category or class seem to be appropriate instead of the misleading type.

Is there a strong preference from anyone?

**Reply by thorsted** [reply]
On PRONOM's site and in conversations with them, I think they will using the "Classification" field. Classification is also used in Rosetta, so that would be my choice.

**Comment by BertrandCaron** [comment]
Personnally, I like the idea of specifying "**format** classification/category". Indeed, the content type might be different from the format type (a digitized text or a spreadsheet preserved as an image, for example).

**Comment by richardlehane** [comment]
this feature implemented in sf 1.10 release, I'll leave this discussion open for a little while for feedback on the feature

**Comment by robin-francois** [comment]
Great work. Gotta test that. Thank you!
